components/raftstore-v2/src/worker/tablet.rs

```diff
@@ -324,9 +389,9 @@ impl<EK: KvEngine> Runner<EK> {
                     }
```

Comment:
At line 388, it returns without invoke callback, it may leak a dirty mark, as a result, snapshot can not be generated.

Reply:
That's by design. It will be retried next restart.